holmes/config.py

```diff
@@ -54,24 +54,55 @@ class SupportedTicketSources(str, Enum):
 
 
 def get_env_replacement(value: str) -> Optional[str]:
-    env_values = re.findall(r"{{\s*env\.([^\s]*)\s*}}", value)
-    if not env_values:
-        return None
-    env_var_key = env_values[0].strip()
-    if env_var_key not in os.environ:
-        msg = f"ENV var replacement {env_var_key} does not exist for param: {value}"
-        logging.error(msg)
-        raise Exception(msg)
+    env_patterns = re.findall(r"{{\s*env\.([^}]*)\s*}}", value)
+
+    # Find all non-env patterns ({{ anything_else }})
+    all_patterns = re.findall(r"{{\s*([^}]*)\s*}}", value)
+    non_env_patterns = [p for p in all_patterns if not p.strip().startswith("env.")]
+
+    if non_env_patterns and not env_patterns:
```

Comment:
Can you elaborate what is this use case? e.g. what is non_env_patterns that we are fine with keeping? when we have such a thing?

Reply:
I changed the specs and simplified the code. There was a bunch of code that was removing any double curly braces unless they were a well formed `{{ env.... }}` string.
Now they are left as-is and only the correct strings are replaced.